apps/coscrad-frontend/src/components/transcripts/participants-presenter.tsx

```diff
@@ -0,0 +1,22 @@
+import { ITranscriptParticipant } from '@coscrad/api-interfaces';
+import { Divider, Typography } from '@mui/material';
+
+interface ParticipantsPresenterProps {
+    participants: ITranscriptParticipant[];
+}
+
+export const ParticipantsPresenter = ({ participants }: ParticipantsPresenterProps) => {
+    return (
```

Comment:
Do you want to use your `if(length==0) return null` trick here?

Reply:
We put it in the parent component because if there are no participants, we don't want to display the `Card` with the "Transcription" title.